personio-personnel-data-api-oa3.yaml

```diff
@@ -203,53 +258,62 @@ paths:
                                 friday: '08:00'
                                 saturday: '00:00'
                                 sunday: '00:00'
+                            type: standard
+                            universal_id: work_schedule
                           fix_salary:
                             label: Fix salary
                             value: 3000
+                            type: decimal
+                            universal_id: fix_salary
                           fix_salary_interval:
                             label: Salary interval
                             value: monthly
+                            type: standard
+                            universal_id: fix_salary_interval
                           hourly_salary:
                             label: Hourly salary
                             value: 0
+                            type: decimal
+                            universal_id: hourly_salary
                           vacation_day_balance:
                             label: Vacation day balance
                             value: 2
+                            type: decimal
+                            universal_id: vacation_day_balance
                           last_working_day:
                             label: Last day of work
                             value:
+                            type: date
+                            universal_id: last_working_day
                           profile_picture:
                             label: Profile Picture
                             value: https://api.personio.de/v1/company/employees/1132888/profile-picture
+                            type: standard
+                            universal_id: profile_picture
                           team:
                             label: Team
                             value:
                               type: Team
                               attributes:
                                 id: 59026
                                 name: T_X
+                            type: standard
+                            universal_id: team
                           dynamic_24407:
                             label: Titel
                             value: Dr
+                            universal_id: null
```

Comment:
Does this even make sense? How can the `universal_id` be null?

Reply:
yes, universal_id could be null